_data/entertainment.yml

```diff
@@ -104,6 +110,8 @@ websites:
     tfa:
       - totp
     doc: https://qa.nicovideo.jp/faq/show/4500
+    regions:
+      - jp
 
   - name: Pandora
```

Comment:
Pandora is [not available outside the US](https://help.pandora.com/s/article/Pandora-in-Other-Languages?language=en_US). I'd suggest adding a US region tag to Pandora.

Reply:
Should be all done now.